packages/styles/scss/components/text-input/_text-input.scss

```diff
@@ -131,6 +131,11 @@
     transition: outline $duration-fast-01 motion(standard, productive);
   }
 
+  .#{$prefix}--toggle-password-tooltip .#{$prefix}--popover {
+    position: absolute;
+    inset-inline-start: -(convert.to-rem(40px));
+  }
```

Comment:
Does this work with all alignment options? Looking at it in the dev tools I don't see `position: absolute` modifying it in any way - could possibly be removed?